### PR TITLE
refactored pynumerals from channumerals; improved tests

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

**Review by chrzyki — APPROVED** [review]
Looks good, thanks!

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+minversion = 3.6
+testpaths = tests
+mock_use_standalone_module = true
+addopts = --cov
+
+[easy_install]
+zip_ok = false
+
+[coverage:run]
+source = pynumerals
```
Comment (xrotwang) [inline setup.cfg]: Please include the `tests` directory in the coverage report. This makes sure we don't accumulate stale/obsolete/nut-run tests.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    include_package_data=True,
+    zip_safe=False,
+    platforms='any',
+    python_requires='>=3.5',
+    install_requires=[
+        'attr',
+        'beautifulsoup4>=4.6.3',
+        'clldutils',
+        'fuzzywuzzy',
+        'pytest',
```
Comment (xrotwang) [inline setup.py]: `pytest` is only required to run tests, right? So it should be required as `extra`, see e.g. https://github.com/pofatu/pygeoroc/blob/cd0d249aeb43c33222d452bc57a9f64b06a82714/setup.py#L37-L46

**Review by xrotwang — CHANGES_REQUESTED** [review]
Please move the package directory into a `src` directory to make sure testing via `tox` can reveal packaging problems, too.

**Review by xrotwang — APPROVED** [review]

**Pull request merged**